src/material-experimental/mdc-form-field/_form-field-subscript.scss

```diff
@@ -16,14 +16,20 @@
 
   .mat-mdc-form-field-hint-wrapper,
   .mat-mdc-form-field-error-wrapper {
-    position: absolute;
-    top: 0;
-    left: 0;
-    right: 0;
     padding: 0 mdc-textfield-variables.$padding-horizontal;
   }
 
-  .mat-mdc-form-field-bottom-align::before {
+  .mat-mdc-form-field-bottom-align:not(.mat-mdc-form-field-subscript-dynamic-size) {
```

Comment:
We'll see how the presubmit goes, but I worry about the specificity increase. An alternate approach would be to add a stronger rule for the dynamic class and set `position: static`

Reply:
If the presubmit fails due to this, I'll change it. I took a quick look and did not see anything that would conflict with it.

Reply:
I did end up having to fix the selectors - passing now